src/GitHub.VisualStudio/UI/Views/Controls/PullRequestListItem.xaml

```diff
@@ -57,7 +57,7 @@
             TextTrimming="CharacterEllipsis"
             Margin="0,-3,5,0"
             Content="{Binding Title}"
-            ToolTip="{Binding Title}"
+            ToolTip="View Pull Request details"
```

Comment:
Thanks @paladique for the links!
https://github.com/github/VisualStudio/blob/396baff13f4cecb0a035f6a7569a23a929652900/src/GitHub.VisualStudio.UI/Resources.resx

Here's an example of how we do it within this project https://github.com/github/VisualStudio/blob/396baff13f4cecb0a035f6a7569a23a929652900/src/GitHub.VisualStudio/UI/Views/Controls/LoginControl.xaml#L87